app/containers/Industry/UserDetail.js

```diff
@@ -0,0 +1,183 @@
+import React, {useState} from 'react';
+import {Jumbotron, Button, Table, Form} from 'react-bootstrap';
+import {graphql, createFragmentContainer} from 'react-relay';
+import editUserMutation from '../../mutations/industry/editUserMutation';
+import {useInput} from './InputHook';
+
+const edit = {
+  width: '75px'
+};
+const table = {
+  marginTop: '30px',
+  marginBottom: '50px'
+};
+
+export const UserDetail = props => {
+  const {user} = props;
+  const [editMode, setMode] = useState(0);
```

Comment:
Why is this a number and not a boolean?

Reply:
i used a number bc its easier to update inline for onclick

Reply:
`setMode(true)` or `setMode(!editMode)` are easy with a boolean.